package.xml

```diff
@@ -1,18 +1,24 @@
 <?xml version="1.0"?>
-<package>
+<package format="2">
   <name>teleop_twist_keyboard</name>
   <version>0.6.0</version>
-  <description>Generic keyboard teleop for twist robots.</description>
+  <license>BSD</license>
+  <description>
+    A robot-agnostic teleoperation node to convert keyboard commands to Twist
+    messages.
+  </description>
 
   <maintainer email="namniart@gmail.com">Austin Hendrix</maintainer>
```

Comment:
If you update the maintainer field make sure to add an author tag to give credit to the current maintainer